### PR TITLE
SkyTemple Beta 1.4.0rc1

### PR DESCRIPTION
**Pull request by theCapypara**

### CODE REVIEW

**Comment by theCapypara** [comment]
bot build org.skytemple.SkyTemple

**Comment by theCapypara** [comment]
bot, build org.skytemple.SkyTemple

File: org.skytemple.SkyTemple.yml [hunk 1]
```diff
@@ -119,13 +119,13 @@ modules:
...
       - type: archive
-        url: https://github.com/SkyTemple/skytemple/archive/refs/tags/1.4.0a1.tar.gz
-        sha256: 57a605463a2401799aa394682f4309de9e538558510de61ce25bc2d534c8d71c
+        url: https://github.com/SkyTemple/skytemple/archive/refs/tags/1.4.0rc1.tar.gz
+        sha256: a4706d76580facfdeb43f127689dda0dd2f2db2d89075ad0a55029478c0fc3ae
       - type: file
         path: run.sh
       - type: file
-        url: https://raw.githubusercontent.com/SkyTemple/skytemple/5263ab591e9d9ccea0e96355d532f4672ddd67dc/org.skytemple.SkyTemple.desktop
+        url: https://raw.githubusercontent.com/SkyTemple/skytemple/1.4.0rc1/org.skytemple.SkyTemple.desktop
```
Comment (hfiguiere) [inline org.skytemple.SkyTemple.yml:127]: aren't these files in the archive now?

Comment (theCapypara) [reply org.skytemple.SkyTemple.yml:127]: Oh right, they should be (and if not I should fix that). I'll make sure to update it to use the files from the archive for the release / next RC.

**Pull request merged**